modules/deployment-model-impl/src/main/java/org/mule/runtime/module/deployment/impl/internal/application/ApplicationDescriptorFactory.java

```diff
@@ -9,7 +9,7 @@
 import static org.mule.runtime.core.api.config.bootstrap.ArtifactType.APP;
 import static org.mule.runtime.deployment.model.api.application.ApplicationDescriptor.DEFAULT_CONFIGURATION_RESOURCE;
 import static org.mule.runtime.deployment.model.api.application.ApplicationDescriptor.DEFAULT_CONFIGURATION_RESOURCE_LOCATION;
-import static org.mule.runtime.deployment.model.api.application.ApplicationDescriptor.MULE_ARTIFACT_JSON_DESCRIPTOR;
```

Comment:
org.mule.runtime.deployment.model.api.application.ApplicationDescriptor.MULE_ARTIFACT_JSON_DESCRIPTOR must be removed